core/src/test/java/io/grpc/NameResolverProviderTest.java

```diff
@@ -133,117 +89,44 @@ public void baseProviders() {
   }
 
   @Test
-  public void getCandidatesViaHardCoded_triesToLoadClasses() throws Exception {
+  public void getClassesViaHardcoded_triesToLoadClasses() throws Exception {
```

Comment:
After discussion offline: we'd actually be okay without this. Being able to have it is NameResolverProvider being a special case.